transformations/src/main/java/com/commit451/coiltransformations/SquareCropTransformation.kt

```diff
@@ -11,10 +10,10 @@ import kotlin.math.max
  */
 class SquareCropTransformation : Transformation {
 
-    override fun key(): String = SquareCropTransformation::class.java.name
+    override val cacheKey: String = SquareCropTransformation::class.java.name
 
-    override suspend fun transform(pool: BitmapPool, input: Bitmap, size: Size): Bitmap {
+    override suspend fun transform(input: Bitmap, size: Size): Bitmap {
         val largerSize = max(input.width, input.height)
```

Comment:
I think this needs to be updated too. `Size` is now composed of two `Dimension` values which can either be a px or `Dimension.Original` value (representing the original size of the image). TLDR this was necessary to support transformations + unbounded width/heights.

I'd change this to:

```kotlin
val largerSize = max(
    input.width.pxOrElse { input.width },
    input.height.pxOrElse { input.height }
)
```